mu-plugins/central-hub/src/config-store/internals.php

```diff
@@ -62,13 +62,22 @@ function _the_store( $store_key = '', $config_to_store = array(), $remove = null
  * Load a configuration from the filesystem, returning its
  * storage key and configuration parameters.
  *
- * @since 1.0.0
- *
  * @param string $path_to_file Absolute path to the config file.
  *
  * @return array
+ * @throws \Exception
+ * @since 1.1.0
+ *
  */
 function _load_config_from_filesystem( $path_to_file ) {
+	if ( is_null( $path_to_file ) ) {
```

Comment:
@rgadon107 This conditional and error is not needed.  It's redundant as PHP will throw an error automatically through the `require`.